### PR TITLE
PLATFORM-4490: log Helios response details

### PR DESCRIPTION
**Pull request by themech**
Log details added in https://github.com/Wikia/helios/pull/430

### CODE REVIEW

File: includes/User.php [hunk 1]
```diff
@@ -506,7 +506,13 @@ public static function newFromToken( \WebRequest $request ) {
 					} else {
 						WikiaLogger::instance()->error( 'Helios user id mismatch', [
 							'sessionUserId' => $sessionUserId,
-							'userId' => $tokenInfo->user_id
+							'heliosResponse' => [
+								'userId' => $tokenInfo->user_id,
+								'accessToken' => $tokenInfo->access_token,
```
Comment (harnash) [inline includes/User.php:511]: I would only log part of the access token (3-5 of first and last characters) and mask all other characters for security reasons.

**Review by harnash — CHANGES_REQUESTED** [review]

File: includes/User.php (hunk 1 above)
Comment (themech) [reply includes/User.php:511]: isn't it the same token that we send from MW to Helios in get parametes to so it is present in access logs and MediaWiki http request logs?

Comment (harnash) [reply includes/User.php:511]: Yes it is but it is not good practice and I would not like to add more places like this. We would need to clean it up eventually but adding more to the already existing ones won't help us.

Comment (themech) [reply includes/User.php:511]: We agreed offline to keep it this way (which is the same as in other places) to make logs search easier during the bug investigation. This log will be removed once we find and fix the culprit.

**Review by harnash — APPROVED** [review]

**Pull request merged**